docs/rackhd/install_os.rst

```diff
@@ -3,6 +3,105 @@ OS Installation Workflow Support
 
 RackHD workflow support installing Operating System automatically from remote http repository.
 
+Configuring RackHD OS Mirrors
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```

Comment:
I think http://rackhd.readthedocs.io/en/latest/rackhd/install_os.html#setting-up-a-windows-os-repo  should be one part of this section.